code/modules/modular_computers/file_system/data.dm

```diff
@@ -1,20 +1,24 @@
-// /data/ files store data in string format.
-// They don't contain other logic for now.
+/// Data files. Doesn't really hold many important functionalities, here for organization.
 /datum/computer_file/data
-	var/stored_data = "" // Stored data in string format.
+	var/do_not_edit = FALSE // Whether the user will be reminded that the file probably shouldn't be edited.
 	filetype = "DAT"
+
+// /data/text files store data in string format.
+// They don't contain other logic for now.
+/datum/computer_file/data/text
+	var/stored_text = "" // Stored data in string format.
+	filetype = "TXT"
```

Comment:
```suggestion
	filetype = "TXT"
	/// Stored data in string format.
	var/stored_text = ""
```
Try to put redefined variables at the top of the variables definition.
Makes it a little bit easier to see which variables are exclusive to this child and not the parent.